.github/workflows/linux-cp310.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
           pip install -U pip setuptools
       - name: Lint
         run: |
+          pip install .[lint]
```

Comment:
why swapping the installation order?

Reply:
Running `pip install`. after `pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu`  will reinstall torch, causing torch version mismatches with torchvision and torchaudio, resulting in compatibility issues.

Reply:
Tested and confirmed this change is unnecessary.